src/model/Computation.js

```diff
@@ -65,7 +67,7 @@ export default class Computation extends Model {
 		this.boundsSensitive = true;
 		this.dirty = true;
 
-		// TODO: computations don't shuffle, but this is a bit hackish
+		// TODO: is there a less hackish way to do this?
 		this.shuffle = undefined;
```

Comment:
Unfortunately noop isn't an option because the shuffler uses presence of the shuffle method to decide how to proceed.